parser/operators_test.go

```diff
@@ -787,6 +787,19 @@ func TestBuiltInFunctions(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "PositionOf",
+			cql: "PositionOf('B','ABC)",
```

Comment:
Looks like this test is failing. I think you need to add a `'` at the end of the second argument here.